README.md

```diff
@@ -52,16 +88,35 @@ The MotorEase code and data has been permanently archived on Zenodo at: (https:/
 
 - In order to load your own images, navigate to the Data folder in the container and delete the existing photos. Use the wget command to download your images into the directory so they may be used. 
   
-<ins>Python Environment: </ins>
+<ins> Build 3: Python Environment: </ins>
```

Comment:
Would you be able to move this closer to the other build options so it flows better?

Reply:
Ya, I moved Build 3 to be right under Build 2